src/main/java/core/basesyntax/LineProcessor.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.handlers.OperationHandler;
+import core.basesyntax.service.handlers.OperationStrategyImpl;
+import java.util.List;
+
+public class LineProcessor {
```

Comment:
```suggestion
public class TransactionProcessor {
```

and lets move this class to service package(same for ReportFormatter)